src/main/resources/lang/default.lang

```diff
@@ -2275,6 +2275,11 @@ transform reasons:
 	unknown: unknown
 	infection: infection, villager infection
 
+# -- Change Reasons --
+experience change reasons:
```

Comment:
Again should mention cooldown

Reply:
unsure what you want with cooldown here, what he did there is only for enums